internal/diffchecker/genaisrc/diffchecker.genai.mjs

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs';
+
+// NOTE FOR USER TO RUN THIS SCRIPT:
+// > cd internal/diffchecker/genaisrc
+// > npx genaiscript run diffchecker.genai.mjs
+
+// Read the prompt from the markdown file
+const prompt_file = 'prompts/amqpdiff.prompt.md';
+const prompt = fs.readFileSync(prompt_file, 'utf-8');
+const log_py = 'amqpproxy-traffic-python.jsonl';
```

Comment:
Just wondering - did the .jsonl extension (rather than .json) help or change things in any way?

Reply:
It did not change anything for the genaiscript. I just set it to .jsonl for the sake of a python script I had going separately.